provider_base/services/couchdb.json

```diff
@@ -8,8 +8,8 @@
         }
     },
     "couch": {
-        "master": false,
         "port": 5984,
+        "mode": "multimaster",
```

Comment:
at some point in the future, we can make the real default for all nodes be plain couch, but this will be a destructive change.